Applied Challenge 5/Home.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Applied Challenge 1</title>
+    <meta charset="utf-8">
+    <link rel="stylesheet" type="text/css" href="_css/Mowen_a5.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.css" />
+    <script src="http://code.jquery.com/jquery-1.11.1.min.js"></script>
+    <script src="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.js"></script>
+    <link rel="stylesheet" type="text/css" href="_css/bistro.css">
+</head>
+<body>
+<!-- Randall Mowen
+     Home.html
+     INFO2439
+     Thoendel
+     10/12/19
+-->
+<div data-role="page" id="homePage">
+    <header>
+    <div data-role="header" data-position="fixed" data-id="header" class="top-header" data-theme="a">
+    <h1>Sage Bistro Home</h1></div>
+    </header>
+
+    <div data-role="main" class="ui-content">
+    <h3>About us</h3>
+    <p>The Sage Student Bistro operates as a customer-focused classroom providing our students an opportunity to refine their culinary, bakery, service and supervisory skills in a setting where the element of time is as important as the food and service. Because this is a class, the Bistro is open when the students are in session. A standard menu is offered, as well as weekly, student designed prix fixe specials during dinner.Find us on <a href="https://www.facebook.com/sagestudentbistro/likes">Facebook</a> for media and additional info.</p>
```

Comment:
I love that you included some about info with the navigation options. Very professional!